src/pymbus/codes/vif.py

```diff
@@ -64,781 +69,822 @@ class VIFCodeUnit(str, Enum):
     ),
     0b0110_0001: VIFCode(
-        code=0x61,
+        # code=0x61,
         coef=1e-2,
-        desc=VIFCodeDescription.temp_difference,
+        kind=VIFCodeKind.temperature_difference,
         unit=VIFCodeUnit.kelvin,
     ),
     0b0110_0010: VIFCode(
-        code=0x62,
+        # code=0x62,
         coef=1e-1,
-        desc=VIFCodeDescription.temp_difference,
+        kind=VIFCodeKind.temperature_difference,
         unit=VIFCodeUnit.kelvin,
     ),
     0b0110_0011: VIFCode(
-        code=0x63,
+        # code=0x63,
         coef=1e0,
-        desc=VIFCodeDescription.temp_difference,
+        kind=VIFCodeKind.temperature_difference,
         unit=VIFCodeUnit.kelvin,
     ),
     # E110_01nn - External temperature (Celsius = C)
     0b0110_0100: VIFCode(
-        code=0x64,
+        # code=0x64,
         coef=1e-3,
-        desc=VIFCodeDescription.external_temp,
+        kind=VIFCodeKind.external_temperature,
         unit=VIFCodeUnit.celsius,
     ),
     0b0110_0101: VIFCode(
-        code=0x65,
+        # code=0x65,
         coef=1e-2,
-        desc=VIFCodeDescription.external_temp,
+        kind=VIFCodeKind.external_temperature,
         unit=VIFCodeUnit.celsius,
     ),
     0b0110_0110: VIFCode(
-        code=0x66,
+        # code=0x66,
         coef=1e-1,
-        desc=VIFCodeDescription.external_temp,
+        kind=VIFCodeKind.external_temperature,
         unit=VIFCodeUnit.celsius,
     ),
     0b0110_0111: VIFCode(
-        code=0x67,
+        # code=0x67,
         coef=1e0,
-        desc=VIFCodeDescription.external_temp,
+        kind=VIFCodeKind.external_temperature,
         unit=VIFCodeUnit.celsius,
     ),
     # E110_10nn - Pressure (bar)
     0b0110_1000: VIFCode(
-        code=0x68,
+        # code=0x68,
         coef=1e-3,
-        desc=VIFCodeDescription.pressure,
+        kind=VIFCodeKind.pressure,
         unit=VIFCodeUnit.bar,
     ),
     0b0110_1001: VIFCode(
-        code=0x69,
+        # code=0x69,
         coef=1e-2,
-        desc=VIFCodeDescription.pressure,
+        kind=VIFCodeKind.pressure,
         unit=VIFCodeUnit.bar,
     ),
     0b0110_1010: VIFCode(
-        code=0x6A,
+        # code=0x6A,
         coef=1e-1,
-        desc=VIFCodeDescription.pressure,
+        kind=VIFCodeKind.pressure,
         unit=VIFCodeUnit.bar,
     ),
     0b0110_1011: VIFCode(
-        code=0x6B,
+        # code=0x6B,
         coef=1e0,
-        desc=VIFCodeDescription.pressure,
+        kind=VIFCodeKind.pressure,
         unit=VIFCodeUnit.bar,
     ),
     # E110_110n - Time point (date or datetime)
     0b0110_1100: VIFCode(
-        code=0x6C,
+        # code=0x6C,
         coef=1,
-        desc=VIFCodeDescription.time_point,
+        kind=VIFCodeKind.time_point,
         unit=VIFCodeUnit.date,
     ),
     0b0110_1101: VIFCode(
-        code=0x6D,
+        # code=0x6D,
         coef=1,
-        desc=VIFCodeDescription.time_point,
+        kind=VIFCodeKind.time_point,
         unit=VIFCodeUnit.datetime,
     ),
```

Comment:
Is it measurable? This is about time points as dates or datetimes, I am not sure it is either physical or measurable.